src/core/a-mixin.js

```diff
@@ -1,38 +1,72 @@
 /* global HTMLElement */
-var AComponents = require('./component').components;
 var ANode = require('./a-node');
 var registerElement = require('./a-register-element').registerElement;
+var components = require('./component').components;
 
+/**
+ * @member {object} attrsCache - Cache of pre parsed component attributes
+ */
 module.exports = registerElement('a-mixin', {
   prototype: Object.create(
     ANode.prototype,
     {
+      createdCallback: {
+        value: function () {
+          this.attrsCache = {};
```

Comment:
Can just call this `attrCache`, that will imply plural.

Though you might want to name this `components` or `componentCache` because these changes will make it so mixins will only work with components. Seems they won't work with any attribute like animations/primitives.